pom.xml

```diff
@@ -76,6 +76,7 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
```

Comment:
I completely disagree :-) Especially for such plugin properties. It allows to change the version of a plugin pre-defined in a parent pom with one line instead of 5 ?! For me the property is a way to make easily configurable stuffs using inheritance. It is also really useful for tests because locally you can just try another version of a plugin (or another config) with just a -DmyProperty=XXX